cinch/roles/jenkins_master/templates/roles_and_ldap_auth.groovy

```diff
@@ -0,0 +1,177 @@
+import hudson.*;
```

Comment:
I'm seeing some mixed indents in this file.  Can we standardize on four spaces?  Perhaps I just don't understand how the parens in groovy affect indentation levels.